src/wp-includes/user.php

```diff
@@ -604,9 +604,19 @@ function wp_validate_logged_in_cookie( $user_id ) {
 function count_user_posts( $userid, $post_type = 'post', $public_only = false ) {
```

Comment:
```suggestion
function count_user_posts( $userid, $post_type = array( 'post' ), $public_only = false ) {
```
Wonder if we should do this?

Reply:
I'm not too fussed, up to you. I have a slight leaning to leaving it as is as it's easier to interpret on [the docs](https://developer.wordpress.org/reference/functions/count_user_posts/) 

If you decide to change it, then the docblock will need an update too.

Reply:
I think it is overkill.